.github/workflows/master.yml

```diff
@@ -1,6 +1,7 @@
 # WARNING: This file is autogenerated - changes will be overwritten if not made via https://github.com/pulumi/ci-mgmt
 
 env:
+  VERSION_PREFIX: '"3.0.0"'
```

Comment:
Ah so we are still using this. It makes sense that it works out then as it constrains pulumictl.

I'm still confused if the plan is to get rid of it or keep using it. It should be relatively easy to get rid of it, then the yml files will not generate any git history churn on major updates.